pytest_httpserver/httpserver.py

```diff
@@ -45,6 +48,43 @@ class NoMethodFoundForMatchingHeaderValueError(Error):
     pass
 
 
+class WaitingSettings:
+    """Class for providing default settings and storing them in HTTPServer
+
+    :param raise_assertions: whether raise assertions on unexpected request or timeout or not
+    :param stop_on_nohandler: whether stop on unexpected request or not
+    :param timeout: time (in seconds) until time is out
+    """
+
+    def __init__(self, raise_assertions: bool = True, stop_on_nohandler: bool = True, timeout: float = 5):
+        self.raise_assertions = raise_assertions
+        self.stop_on_nohandler = stop_on_nohandler
+        self.timeout = timeout
+
+
+class Waiting:
+    """Class for HTTPServer.wait context manager
+
+    This class should not be instantiated directly."""
+
+    def __init__(self):
+        self._result = None
+        self.start = datetime.now()
+        self.stop = None
+
+    def complete(self, result: bool):
+        self._result = result
+        self.stop = datetime.now()
+
+    @property
+    def result(self) -> bool:
+        return self._result
+
+    @property
+    def time(self) -> timedelta:
```

Comment:
Could you change this to "elapsed_time"? I think that would more emphasize what it has (elapsed time), I think "time" itself has no meaning.